### ISSUE TITLE
Most Read Component Accessibility Swarm

### ISSUE THREAD

**Issue by sareh** [BBC-archive/psammead#1536]
**Is your feature request related to a problem? Please describe.**
Once the Most Read component has been created, we should conduct an accessibility swarm.

Full details here: https://bbc.github.io/accessibility-news-and-you/accessibility-swarms 

If there are any bugs that are found, we should raise issues when relevant and prioritise fixing them accordingly.

**Describe the solution you'd like**
- People from all disciplines should take part in the swarm (dev, test, UX, product, BA).
- The results of the swarm should be summarised on this ticket
- Any bugs found should have relevant issues opened. 

- Only after all high priority accessibility bug tickets have been addressed should we change the versioning from `alpha` to a standard whole version number. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Notes here: https://bbc.github.io/accessibility-news-and-you/accessibility-swarms

**Comment by EinsteinNjoroge** [comment]
This will be handled in https://github.com/bbc/psammead/issues/562
